esp_jpeg/test_apps/main/test_usb_camera_jpg.h

```diff
@@ -0,0 +1,175 @@
+// JPEG encoded frame 160x120, 2632 bytes, no huffman tables, double block size (16x8 pixels)
+
+/*
+This is raw data from Logitech C270 USB camera.
+It was reconstructed to usb_camera.jpg file using hex_to_jpg.py script
+The it was converted to RGB888 array with jpg_to_rgb888_hex.py
+*/
+
+const unsigned char jpeg_no_huffman[] = {
```

Comment:
Instead of doing conversion from jpg files to C header, you can simply include the jpg file into the binary. PTAL at https://github.com/espressif/idf-extra-components/blob/ca9569400ed009151f3def6117a3c02f85d0e96b/libpng/test_apps/main/CMakeLists.txt#L5 and https://github.com/espressif/idf-extra-components/blob/ca9569400ed009151f3def6117a3c02f85d0e96b/libpng/test_apps/main/test_libpng.c#L12-L13